### PR TITLE
Updated test.c code in documentation to fix issue with existing code …

### PR DESCRIPTION
**Pull request by justin-echternach**
…in doc ...

Signed-off-by: Justin Echternach <justin.echternach@positivespaces.io>

### CODE REVIEW

File: docs/book/en/src/contribute/build_from_src/android/ndk.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ int main(int argc, char *argv[]) {
   WasmEdge_ConfigureDelete(ConfCxt);
   
   /* Create the image and TFLite import objects. */
-  WasmEdge_ImportObjectContext *ImageImpObj = WasmEdge_Image_ImportObjectCreate();
-  WasmEdge_ImportObjectContext *TFLiteImpObj = WasmEdge_TensorflowLite_ImportObjectCreate();
-  WasmEdge_ImportObjectContext *TFDummyImpObj = WasmEdge_Tensorflow_ImportObjectCreateDummy();
+  WasmEdge_ModuleInstanceContext *ImageImpObj = WasmEdge_Image_ImportObjectCreate();
```
Comment (q82419) [inline docs/book/en/src/contribute/build_from_src/android/ndk.md:50]: Should be `WasmEdge_Image_ModuleInstanceCreate()`.

File: docs/book/en/src/contribute/build_from_src/android/ndk.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ int main(int argc, char *argv[]) {
...
+  WasmEdge_ModuleInstanceContext *TFLiteImpObj = WasmEdge_TensorflowLite_ImportObjectCreate();
```
Comment (q82419) [inline docs/book/en/src/contribute/build_from_src/android/ndk.md:51]: Should be `WasmEdge_TensorflowLite_ModuleInstanceCreate()`.

File: docs/book/en/src/contribute/build_from_src/android/ndk.md [hunk 3]
```diff
@@ -47,9 +47,9 @@ int main(int argc, char *argv[]) {
...
+  WasmEdge_ModuleInstanceContext *TFDummyImpObj = WasmEdge_Tensorflow_ImportObjectCreateDummy();
```
Comment (q82419) [inline docs/book/en/src/contribute/build_from_src/android/ndk.md:52]: Should be `WasmEdge_Tensorflow_ModuleInstanceCreateDummy()`.

**Review by q82419 — CHANGES_REQUESTED** [review]
Nice catch. This doc used the `0.9.1` APIs and should be updated.
Please help to fix the commented codes.

File: docs/book/en/src/contribute/build_from_src/android/ndk.md (hunk 1 above)
Comment (justin-echternach) [reply docs/book/en/src/contribute/build_from_src/android/ndk.md:50]: oops ... forgot to copy that to my repo ... will do

**Comment by justin-echternach** [comment]
I think I got it ... let me know if there are any more issues

**Review by q82419 — APPROVED** [review]

**Review by hydai — APPROVED** [review]

**Comment by hydai** [comment]
@justin-echternach 
Thanks for your contribution :-)

**Pull request merged**